src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborDataItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal enum CborMajorType : byte
+    {
+        PositiveInteger = 0,
```

Comment:
In hindsight, calling it `PositiveInteger` is technically correct, since it includes zero. Will change to `UnsignedInteger`.